### DISCUSSION TITLE
How to disable jdtls messages at lower right when writing java code?

### DISCUSSION THREAD

**Discussion by santosh** [LazyVim/LazyVim#3044] [Q&A] [answered]
I have an existing lazyvim installation which I've been using to write code in go and python.

Today I enabled support for java by writing this line:

```
require("lazy").setup({
  spec = {
    -- other configs....
    { import = "lazyvim.plugins.extras.lang.java" },
    -- other configs...
  },
```

After opening the a java file for the first time, it installed jdtls. Which seems like a linter for Java.

The issue is that it shows a "Validating document" on every keystroke. 

I don't if it should check on every keystroke or not. But the messages should not be coming at every keystroke. Either a message at certain timeout, or no messages at all (like other language integration).

How can I deal with this messages as I'm typing?

**Comment by rogeriofrsouza** [comment, marked as answer]
This discussion may help you https://github.com/LazyVim/LazyVim/discussions/1439
